packages/suite/src/actions/suite/constants/metadataConstants.ts

```diff
@@ -36,3 +36,5 @@ export const GOOGLE_IMPLICIT_FLOW_CLIENT_ID =
 
 // dropbox allows authorization code flow for both web and desktop without client secret
 export const DROPBOX_CLIENT_ID = 'wg0yz2pbgjyhoda';
+
+export const ENCRYPTION_VERSION = 1;
```

Comment:
I have it as `export const METADATA_ENCODING_VERSION: EncodingVersion = 2;`, perhaps would make sense for one of us to change it so it's the same?

Reply:
I vote for encryption.

```
"Encoding data is a process involving changing data into a new format using a scheme. Encoding is a reversible process and data can be encoded to a new format and decoded to its original format. Encoding typically involves a publicly available scheme that is easily reversed. Encoding data is typically used to ensure the integrity and usability of data and is commonly used when data cannot be transferred in its current format between systems or applications. Encoding is not used to protect or secure data because it is easy to reverse.

An example of encoding is: Base64
```

Reply:
Sure, I'll change it then :) Will you add the type?